index.js

```diff
@@ -152,6 +161,41 @@ module.exports.processV2 = (output, origin) => {
   throw 'no url';
 };
 
+module.exports.processV3 = (output, origin) => {
+  const data = JSON.parse(output.toString().trim());
+  const { automatic_captions, formats, fragments, subtitles, url: audio } = data;
+
+  const cookies = data.http_headers && data.http_headers.Cookie || '';
+  const duration = data.duration || 0;
+  const subtitleFile = findBestSubtitleFile(subtitles) || findBestSubtitleFile(automatic_captions);
```

Comment:
Are you doing this because `subtitles` could be undefined or null? You probably shouldn't do it like this because you're evaluating the return value of `findBestSubtitleFile` instead of `subtitles`, unless of course this is what the intent is?

Reply:
Not sure what these two lines of code are meant to do... but this is identical to what the `process` and `processV2` function does. 

`processV3` is a whole new function, but these lines of code are unchanged from the previous two versions.

Reply:
(essentially, subtitle handling hasn't changed since process v1)

Reply:
Yeah that's fine I'm just asking what the intention is, as I'm not sure if it seems idiomatic or not

Reply:
i just used the same logic for selecting the subtitle file as the V2 method. No reason to change it.